### PR TITLE
PrimeFactorsOfNumbers.java

### PR DESCRIPTION
**Pull request by Mayshank18**
Prime factors of the numbers

# Description

A java program to find the Prime factors of a number

Fixes #(438)

## Type of change

A new Java program in the topic mentioned above assigned to me

# Checklist:

-   [x] I have read the contribution guidelines before making this PR.
-   [x] My code follows the style guidelines of this project
-   [x] I have performed a self-review of my own code
-   [x] I have commented in my code, particularly in hard-to-understand areas
-   [ ] I have made corresponding changes to the documentation (Language folder or any other readme instructed to)
-   [x] My changes generates no new warnings
-   [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by kaustubhgupta** [comment]
The corresponding issue is not linked to this PR

File: Java/PrimeFactorsOfNumbers.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
+// Program to print all prime factors
```
Comment (kaustubhgupta) [inline Java/PrimeFactorsOfNumbers.java:1]: update java folder readme

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

**Review by YashSaxena9 — COMMENTED** [review]
Your code is plagiarised

**Comment by YashSaxena9** [comment]
It seems like you have cheated and copied your code from GFG website. Even your comments match word to word.
We don't accept contributions with plagiarised content.

**Pull request closed without merging**